tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "include": ["env.d.ts", "**/*.ts", "**/*.tsx"],
   "compilerOptions": {
-    "lib": ["DOM", "DOM.Iterable", "ES2019"],
+    "lib": ["DOM", "DOM.Iterable", "ES2019", "ES2022"],
```

Comment:
I'm fine with updating the `"lib"`, but we don't need to have `"ES2019"` and `"ES2022"`